### DISCUSSION TITLE
Skiovox patched on 119?

### DISCUSSION THREAD

**Discussion by brobcoolguy** [3kh0/ext-remover#888] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
used the skiovox exploit on 116 and it worked fine, sadly i forgot to do CAUB so my chromebook updated to 119 😞, now when i try to do
skiovox 2a: pressing esc really fast no longer works (the back button is gone in wifi screen)
skiovox 2b: the link to settings opens up some different settings tab that doesnt unblock anything
skiovox 2c: chromevox was disabled from being turned on using the shortcut, atleast in any kiosks modes

i tried all the kiosk apps on my chromebook and it looks like its patched. are there any exploits or anything i can still do on 119 now or is there nothing i can do? (ps: my school uses lightspeed filter agent to block stuff, if that helps)

**Comment by PenguinLord561** [comment]
@brobcoolguy yes it is sadly patched- im in the same boat as you, i have no longer any exploit to unblock long term. (ltmeat still works but only works for 3 minutes) you should caub just in case a exploit is found with 119

**Reply by brobcoolguy** [reply]
yea same for me, ltmeat works but i can never get it to last very long. fortunately, i figured out that my school has not blocked chrome remote desktop so i can access my PC from my chromebook :)

**Reply by AshtonDavies** [reply]
How do you keep it connected for a long period of time?

**Reply by funkymokey** [reply]
this is genius ill try this with my pc

**Reply by AshtonDavies** [reply]
I mean, I guess it works at school if you leave it running for a bit until it prompts your PC to keep the support session going. Dismissing that prompt once should allow the connection to last longer I believe. Then you just gotta keep your Chromebook online and connected to your PC on your way to School. Difficult process, which is why I wouldn't recommend attempting Chrome Remote Desktop for that purpose.

That's based on the last time I used Chrome Remote Desktop. It could have changed since then.

**Reply by brobcoolguy** [reply]
for me i just sign in to the same account on my pc and chromebook, then set up my pc as a remote device, so then I just keep my PC on and i can access it just by entering a pin.

**Reply by CGRmags** [reply]
@brobcoolguy How do you do Skiovox on 120 pls help I have the same probelm

**Comment by B06e** [comment]
is crosh unblocked?

**Reply by PenguinLord561** [reply]
@B06e No, crosh is unfortunately now blocked-

**Reply by leahanderson1** [reply]
depends on your school policy ackthually 🤓 ☝🏻

**Reply by PenguinLord561** [reply]
@OtterCodes101 Sorry, I wasn't clear. I should of said "No, crosh is unfortunately now blocked for me-"

**Comment by Su4o** [comment, marked as answer]
If you are ChromeOS 119, there's still SKIOVOX for you! (thank @Fighterhen63, not me)

Yes this is a solution, not a discussion --> https://github.com/bypassiwastaken/skiovox-helper/issues/106

**Reply by DankFireR** [reply]
ngl i was boutta make a point about you not reading the post then i read that was just a fix lol, good fix tho

**Reply by Gaboom63** [reply]
Yeah if you have any question just let me know!

**Reply by brobcoolguy** [reply]
yoooo thx!!! i was missing skiovox so much so glad to have it back

**Reply by Aka-but-nice** [reply]
You know... you could have just pinged me? I developed most of the methods for it but ok.

**Reply by Su4o** [reply]
mb

**Reply by AshtonDavies** [reply]
Bro thinks he's vk6. 😭😭😂

**Reply by Su4o** [reply]
💀

**Reply by Aka-but-nice** [reply]
@AshtonDavies  :skull:?

**Reply by AshtonDavies** [reply]
> @AshtonDavies  :skull:?

You did not develop most SKIOVOX methods. That's a fat lie.

**Reply by CGRmags** [reply]
@AshtonDavies Could you see if there is anyone to help me with SKIOVOX 120 pls

**Reply by Su4o** [reply]
contact @Fighterhen63 for help @CGRmags

**Reply by CGRmags** [reply]
@Su4o how?

**Reply by Aka-but-nice** [reply]
Whaa?? When I first discovered the exploit I literally sat there discovering alternate methods for it, the new ones you might see are just built off of the methods I found, credit Bypassi as well.

**Reply by funkymokey** [reply]
Aka-but-nice if you can find exploits like that make yourself known fr

**Reply by shimobri** [reply]
Dunno if anyone is online, but just double check in the case we did smth wrong before, is Skiovox dead at Chrome 121? It still works back in 120 then school update to Chrome 125, we somehow manage to get a Chromebook that's Chrome 121, so asking if it works or not.

**Reply by AshtonDavies** [reply]
> Dunno if anyone is online, but just double check in the case we did smth wrong before, is Skiovox dead at Chrome 121? It still works back in 120 then school update to Chrome 125, we somehow manage to get a Chromebook that's Chrome 121, so asking if it works or not.

Yeah it's been dead since v121, except it still works for some depending on whether multiple user sign-in is disabled or not.

It initially got me thinking you all should just forget SKIOVOX, but SKIOVOX unlocks capabilities beyond web browsing and extension installation.

**Comment by CGRmags** [comment]
@Fighterhen63 Can you help me get SKIOVOX on 120?

**Reply by Gaboom63** [reply]
Yeah! Go ahead and look at this thread: https://github.com/bypassiwastaken/skiovox-helper/issues/106 
I have noticed that dragging has stopped working, but that might just be my chromebook. Let me know if you need anything!

**Comment by CGRmags** [comment]
I found out how to do it anyway. Can I work with you on a simplified guide
to do it?

On Thu, Feb 1, 2024 at 5:51 PM Fighterhen63 ***@***.***>
wrote:

> Yeah! Go ahead and look at this thread: bypassiwastaken/skiovox-helper#106
> <https://github.com/bypassiwastaken/skiovox-helper/issues/106>
> I have noticed that dragging has stopped working, but that might just be
> my chromebook. Let me know if you need anything!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/888#discussioncomment-8341161>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBCGQNZ2DY2TMAPMTE3YRRBDHAVCNFSM6AAAAABCH7SHU2VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGNBRGE3DC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.
